### PR TITLE
Update extension setup

### PR DESCRIPTION
**Pull request by gingermusketeer**
This fixes the debug task which allows the extension to be more easily run and debugged.

This was updated by running `npm install -g yo generator-code` and then `yo code`. After this the files from the new extension were copied into this repo and merged with what was already present.

### CODE REVIEW

**Comment by gingermusketeer** [comment]
@bessey Hopefully this is useful!

File: src/extension.ts [hunk 1]
```diff
@@ -1,19 +1,16 @@
-"use strict";
-import { languages } from "vscode";
+import * as vscode from "vscode";
```
Comment (bessey) [inline src/extension.ts]: Is this better than importing `{languages}`?

**Review by bessey — COMMENTED** [review]
Thanks for doing this, I did wonder if it was possible. Just the one comment but the rest seems good to merge

File: src/extension.ts (hunk 1 above)
Comment (gingermusketeer) [reply src/extension.ts]: `import * as vscode from "vscode";` is what is present by default when generating a new extension.

Have changed it back as it is potentially better when bundling to only import what is needed.

**Comment by bessey** [comment]
Thanks for making the changes! LGTM

**Pull request merged**